cfme/fixtures/pytest_selenium.py

```diff
@@ -53,6 +58,11 @@
 urls = []
 
 
+removed_module(__name__, removal_version="framework 3.0")
```

Comment:
Would it make sense here to have a "framework3.0" removal decorator, but one which you can pass in the "message", it can then be defined somewhere a little more common.